spec/features/shared/header_spec.rb

```diff
@@ -107,39 +106,39 @@
       it 'should redirect to XR-Network index page' do
         click_on 'XR Network'
         click_on 'Community'
-        expect(page).to have_current_path('/communities/xr-network')
+        expect(page).to have_current_path('/communities/va-immersive')
```

Comment:
Keeping in mind that the XR Network links will be changed it seemed logical to make only the minimum changes to these specs in the interest of this pr. I would imagine this will be integrated into the work to rename / remove XR Network so hopefully this will make for a simple rebase.